login-workflow/example/src/screens/demo-components/CreateNewOrgScreenBaseDemo.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { CreateNewOrgScreenBase } from '@brightlayer-ui/react-auth-workflow';
+
+export const CreateNewOrgScreenBaseDemo: React.FC = () => (
+    <CreateNewOrgScreenBase
+        WorkflowCardHeaderProps={{ title: 'Create An Organization' }}
+        WorkflowCardInstructionProps={{
+            instructions:
+                'Enter your organization name to continue with account creation. Fields marked with an (*) are required.',
```

Comment:
There is only one field on the page and it is required. Per our design guidance, let's leave out this second sentence and leave out the asterisk from the input field.